packages/core/src/storage.ts

```diff
@@ -23,6 +23,7 @@ export type DocumentQuery<T extends Doc> = Partial<Data<T>>
 export interface Storage {
   findAll<T extends Doc>(_class: Ref<Class<T>>, query: DocumentQuery<T>): Promise<T[]>
   tx(tx: Tx): Promise<void>
+  isDerived(_class: Ref<Class<Obj>>, _extendedClass: Ref<Class<Obj>>): boolean
```

Comment:
I would love to have this parametrized by `<T extends Obj>`, so at least users can not supply "is Obj extends Person".

Another thing is I do not understand parameters meaning: if I want to check that 'Student extends Person" should I `isDerived(Person, Student)` or `isDerived(Student, Person`? I suggest to rename second parameter to `from`, so it will be clear `isDerived(Student, Person) === true`